static/src/js/containers/ShapefileDropzoneContainer/ShapefileDropzoneContainer.jsx

```diff
@@ -85,13 +85,24 @@ export const ShapefileDropzoneContainer = ({
     }
     onError={
       (file) => {
-        onToggleShapefileUploadModal(false)
+        let shapefileError = ''
 
-        if (file.name.match('.*shp')) {
-          onShapefileErrored({
-            type: 'upload_shape'
-          })
+        onToggleShapefileUploadModal(false)
+        if (file.name.match('.*zip|shp|dbf|shx')) {
```

Comment:
This regex only tests that those letters show up somewhere within the filename, not that the filename ends in those letters

This first example shouldn't return a value because .txt should be an invalid error
<img width="503" alt="Screenshot 2024-11-06 at 1 44 14 PM" src="https://github.com/user-attachments/assets/3d8300bf-39d2-4a8f-8790-cb0e2f1bfe17">

This regex ensures that one of the extensions we are looking for shows up at the end of the string (`$`) after a period (`\.`), and any amount of characters showing up before the extension (`.*`)
```.*\.(zip|shp|dbf|shx)$```